tensorflow/core/debug/debug_grpc_testlib.cc

```diff
@@ -56,17 +56,15 @@ ::grpc::Status TestEventListenerImpl::SendEvents(
 
       // Obtain the device name, which is encoded in JSON.
       third_party::tensorflow::core::debug::DebuggerEventMetadata metadata;
-      for (int i = 0; i < val.metadata().plugin_data_size(); i++) {
-        if (val.metadata().plugin_data(i).plugin_name() != "debugger") {
-          // This plugin data was meant for another plugin.
-          continue;
-        }
-        auto status = tensorflow::protobuf::util::JsonStringToMessage(
-            val.metadata().plugin_data(i).content(), &metadata);
-        if (status.ok()) {
-          // The device name has been determined.
-          break;
-        }
+      if (val.metadata().plugin_data().plugin_name() != "debugger") {
+        // This plugin data was meant for another plugin.
+        continue;
+      }
+      auto status = tensorflow::protobuf::util::JsonStringToMessage(
+          val.metadata().plugin_data().content(), &metadata);
+      if (!status.ok()) {
+        // The device name could not be determined.
```

Comment:
@caisq We might want to log a warning here here. However we shouldn't make that change in this PR.